lcgs-examples/fischer/1/p1-eventually-reach-CS-TRUE-01.atl

```diff
@@ -0,0 +1 @@
+<<p0,p1>> F (p0_in_cs)
```

Comment:
Not equivalent to the json version. The number of players varies.

Reply:
Should be fixed in b46fa72